airflow/www/views.py

```diff
@@ -4721,7 +4734,9 @@ def action_mulduplicate(self, connections, session: Session = NEW_SESSION):
 
             potential_connection_ids = [f"{base_conn_id}_copy{i}" for i in range(1, 11)]
 
-            query = session.query(Connection.conn_id).filter(Connection.conn_id.in_(potential_connection_ids))
+            query = session.execute(
```

Comment:
We can use scalars here then in line 4741 do:
```python
found_conn_id_set = {conn_id for conn_id in query}
```
That's, remove the comma

Reply:
done